routes/events.js

```diff
@@ -12,5 +12,6 @@ router.patch("/", authenticate, eventsValidator.updateEvent, events.updateEvent)
 router.patch("/end", authenticate, eventsValidator.endActiveEvent, events.endActiveEvent);
 router.post("/:id/peers", authenticate, eventsValidator.addPeerToEvent, events.addPeerToEvent);
 router.patch("/:id/peers/kickout", authenticate, eventsValidator.kickoutPeer, events.kickoutPeer);
+router.post("/:id/codes", authenticate, eventsValidator.generateEventCode, events.generateEventCode);
```

Comment:
Are we adding `super-user` check here or it will be done in another story?

Reply:
In other PR